Core/Core.cpp

```diff
@@ -251,36 +255,26 @@ void Core_SingleStep() {
 static inline void CoreStateProcessed() {
 	if (coreStatePending) {
 		coreStatePending = false;
-		m_hInactiveEvent.notify_one();
+		m_InactiveCond.notify_one();
 	}
 }
 
-// Some platforms, like Android, do not call this function but handle things on their own.
-void Core_Run(GraphicsContext *ctx)
-{
-#if defined(_DEBUG)
+// Many platforms, like Android, do not call this function but handle things on their own.
+// Instead they simply call NativeRender and NativeUpdate directly.
+void Core_Run(GraphicsContext *ctx) {
```

Comment:
Does Qt not call this function anymore?  I think that's why we had all the ifdefs, so its debugger would work, or something...

I guess we no longer use Qt on mobile?

-[Unknown]

Reply:
It doesn't. And yeah, we no longer support any Qt-on-mobile platform so we could clean up a bit more...